slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -128,7 +132,7 @@ public boolean maySignAttestation(
                   validatorId,
                   signedAttestationsDao);
 
-          if (!attestationValidator.sourceGreaterThanTargetEpoch()) {
+          if (attestationValidator.sourceGreaterThanTargetEpoch()) {
```

Comment:
Nice catch, can a test case be added for this. Thought we had one in DbSlashingProtectionTest but I guess we didn't after all.

Reply:
we did indeed - but the logic was backwards in DbSlashingProtection - so, everything was safe, just theoretically named badly.